test/exceptions/bad-try.txt

```diff
@@ -0,0 +1,23 @@
+;;; EXE: %(wast2wasm)s
```

Comment:
You can just use `;;; TOOL: wast2wasm` (see the top of run-tests.py for the list of valid tools)

Or you can just omit, since wast2wasm is the default. Though that's historical (since wast2wasm used to be the only tool) so maybe it's better to be explicit. :-)